syntaxes/populate-syntax.js

```diff
@@ -39,7 +39,12 @@ const programmingLanguages = [
     ['shell', 'sh', 'bash', 'zsh', 'bashrc', 'bash_profile'],
     ['cs', 'csharp', 'c#'],
     ['fs', 'fsharp', 'f#'],
-    ['dart']
+    ['dart'],
+    ['nim'],
+    ["elixir"],
+    ["erlang"],
```

Comment:
according to erlang grammar, here are the extension used
```suggestion
    ['erlang', 'erl', 'hrl'],
```